docs/general/rendering.md

```diff
@@ -155,7 +155,7 @@ PR please :)
 The resource root is derived from the class name of the resource being serialized.
 e.g. `UserPostSerializer.new(UserPost.new)` will be serialized with the root `user_post` or `user_posts` according the adapter collection pluralization rules.
 
-Specify the root by passing it as an argument to `render`. For example:
+When using the JSON adapter in your initializer (ActiveModelSerializers.config.adapter = :json), you can specify the root by passing it as an argument to `render`. For example:
```

Comment:
> When using the JSON adapter in your initializer (ActiveModelSerializers.config.adapter = :json)

You can also use `render adapter: :json` to specify the adapter, so I think `in your initializer (ActiveModelSerializers.config.adapter = :json)` might be misleading here.

Maybe it would be more clear to start the section by stating that `root` only applies to `JSON` adapter?